samples/minimum_criteria.rst

```diff
@@ -0,0 +1,20 @@
+.. _definition_and_minimum_criteria:
+
+Definition & Minimum Criteria
+#############################
+
+A sample is an application in Zephyr that briefly showcases how to use a feature, subsystem, or domain.
+
+===========================================
+Minimum Requirements/ Criteria for Samples:
+===========================================
+
+* Samples must not use Zephyr's testing framework.
+* Leveraged in CI for quality confidence, its primary purpose is to exemplify the usage of subsystem, feature, or domain.
+
+  * *Note: Must not serve the purpose of a testcase.*
+
+* Must have README clearly explaining the purpose of the sample and its HW requirements and giving sample output, if applicable.
```

Comment:
Here I would expand on that and provide a template and examples of how to procide such READMEs and how to integrate them in the docs.